### ISSUE TITLE
No warning message if unittest file's path to rule files are wrong

### ISSUE THREAD

**Issue by hvnsweeting** [VictoriaMetrics/VictoriaMetrics#7663]
### Describe the bug

When I run vmalert-tool rule unittest, if a path to rule file is wrong, I expect some error message show up to help troubleshooting. Currently there is no message.

### To Reproduce

Follow https://docs.victoriametrics.com/vmalert-tool/
Create test.yaml

```
rule_files:
  - rules.yaml

evaluation_interval: 1m

tests:
  - interval: 1m
    input_series:
      - series: 'up{job="prometheus", instance="localhost:9090"}'
        values: "0+0x1440"

    metricsql_expr_test:
      - expr: subquery_interval_test
        eval_time: 4m
        exp_samples:
          - labels: '{__name__="subquery_interval_test", cluster="prod", instance="localhost:9090", job="prometheus"}'
            value: 1

    alert_rule_test:
      - eval_time: 2h
        groupname: group1
        alertname: InstanceDown
        exp_alerts:
          - exp_labels:
              job: prometheus
              severity: page
              instance: localhost:9090
              cluster: prod
            exp_annotations:
              summary: "Instance localhost:9090 down"
              description: "localhost:9090 of job prometheus in cluster prod has been down for more than 5 minutes."

      - eval_time: 0
        groupname: group1
        alertname: AlwaysFiring
        exp_alerts:
          - exp_labels:
              cluster: prod

      - eval_time: 0
        groupname: group1
        alertname: InstanceDown
        exp_alerts: []
```

Create alerts.yaml

```
# This is the rules file.

groups:
  - name: group1
    rules:
      - alert: InstanceDown
        expr: up == 0
        for: 5m
        labels:
          severity: page
        annotations:
          summary: "Instance {{ $labels.instance }} down"
          description: "{{ $labels.instance }} of job {{ $labels.job }} in cluster {{ $externalLabels.cluster }} has been down for more than 5 minutes."
      - alert: AlwaysFiring
        expr: 1

  - name: group2
    rules:
      - record: job:test:count_over_time1m
        expr: sum without(instance) (count_over_time(test[1m]))
      - record: subquery_interval_test
        expr: count_over_time(up[5m:])
```

Run unittest
```
Unit Testing:  test.yaml
  FAILED

failed to run unit test for file "test.yaml": 
[
    groupname: group1, alertname: AlwaysFiring, time: 0s, 
        exp:[
            0:
              Labels:{alertgroup="group1", alertname="AlwaysFiring", cluster="prod"}
              Annotations:{}
            ], 
        got:[]  
    groupname: group1, alertname: InstanceDown, time: 2h0m0s, 
        exp:[
            0:
              Labels:{alertgroup="group1", alertname="InstanceDown", cluster="prod", instance="localhost:9090", job="prometheus", severity="page"}
              Annotations:{description="localhost:9090 of job prometheus in cluster prod has been down for more than 5 minutes.", summary="Instance localhost:9090 down"}
            ], 
        got:[]  
    expr: "subquery_interval_test", time: 4m0s,
        exp: {__name__="subquery_interval_test", cluster="prod", instance="localhost:9090", job="prometheus"} 1E+00
        got: nil]2024/11/26 22:07:10 unittest failed

```
FAILED without any message telling that there is no `  - rules.yaml` file (note the alerts.yaml file, not rules.yaml).

### Version

vmalert-tool version vmalert-tool-20241126-130623-heads-master-0-g9cfdbc582

### Additional information

The code that parsing config file does have a log message, but it log with warn level, in unittest, which use ERROR level, thus the log not show. This is what I tried to fix via https://github.com/VictoriaMetrics/VictoriaMetrics/pull/7601 but maybe other solutions would work better.
The https://github.com/VictoriaMetrics/VictoriaMetrics/pull/7627 is not related to the problem.

**Comment by Haleygo** [comment]
The fix https://github.com/VictoriaMetrics/VictoriaMetrics/pull/7627 will be included in the next release.

**Comment by hvnsweeting** [comment]
could you help to take another look, since this issue is NOT related to #7627 which fix `if no test file is found`
this issue is about path to rule file in test file is wrong.

**Comment by hvnsweeting** [comment]
Also as mentioned in the description, the code does emit a warn log [here](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/466cbee43304cd81285788aec302ae28a8ab4dad/app/vmalert/config/config.go#L258-L260) but due to ERROR log level of unittest, it shows nothing.

**Comment by Haleygo** [comment]
> could you help to take another look, since this issue is NOT related to #7627 which fix `if no test file is found` this issue is about path to rule file in test file is wrong.

oh, sorry, I misread the description.

**Comment by f41gh7** [comment]
The bugfix was included into [v1.107.0](https://docs.victoriametrics.com/changelog/#v11070)
